x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/details/execution_log_table/execution_log_table.tsx

```diff
@@ -157,19 +227,26 @@ const ExecutionLogTableComponent: React.FC<ExecutionLogTableProps> = ({
     [refetch]
   );
 
-  const onSearchCallback = useCallback((updatedQueryText: string) => {
-    setQueryText(updatedQueryText);
-  }, []);
+  const onSearchCallback = useCallback(
+    (updatedQueryText: string) => {
+      setQueryText(updatedQueryText);
+    },
+    [setQueryText]
+  );
 
-  const onStatusFilterChangeCallback = useCallback((updatedStatusFilters: string[]) => {
-    setStatusFilters(
-      updatedStatusFilters.length ? updatedStatusFilters.sort().join(',') : undefined
-    );
-  }, []);
+  const onStatusFilterChangeCallback = useCallback(
+    (updatedStatusFilters: RuleExecutionStatus[]) => {
+      setStatusFilters(updatedStatusFilters);
+    },
+    [setStatusFilters]
+  );
 
   const onFilterByExecutionIdCallback = useCallback(
```

Comment:
Hm, I'm getting an error when trying to filter alerts by execution ID:

![Screenshot 2022-05-02 at 17 21 13](https://user-images.githubusercontent.com/1938181/166260317-dd406d32-ac86-47be-b1b5-845d2357ba9a.png)

@spong Do you know why that could happen?